packages/native-stack/src/utils/AnimatedHeaderHeightContext.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import type { Animated } from 'react-native';
+
+export const AnimatedHeaderHeightContext = React.createContext<
```

Comment:
Personally, I'd put it next to the hook and not export that context at all

Reply:
I thought about adding an ability for users to use Consumer from the context, but tbh I don't know any specific reason why it wouldn't be possible to just use the value from the hook 😛 

Resolved in https://github.com/react-navigation/react-navigation/pull/11663/commits/c1aa5fef1ef2d7cf15898baf611367da094f98a2, https://github.com/react-navigation/react-navigation/pull/11663/commits/e107aa10f940ba57a31f1a2806d3b1ebddccbf3d.